internal/dispatch/keys/hasher_ristretto.go

```diff
@@ -0,0 +1,72 @@
+//go:build !wasm
+// +build !wasm
+
+package keys
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/dgraph-io/ristretto/z"
```

Comment:
These imports should be together, they're both 3rd party.

Reply:
Damned VSCode...